src/themes/bankrate/theme.json

```diff
@@ -236,6 +240,16 @@
   },
 
   "compounds": {
+    "pros-and-cons": {
+      "title": {
+        "fontFamily": "helvetica",
+        "lineHeight": "md"
+      },
+      "item": {
+        "fontFamily": "helvetica",
```

Comment:
We should just use the `body` font family from the theme, rather than introduce new typefaces

Reply:
According to design font should be Helvetica, but body in bankrate is 'Roboto, -apple-system, system-ui, system-ui, Arial, sans-serif', that's why I introduced new typeface

Reply:
I imagine that's just a failing on the design system. We should keep body styles consistent - if they should be helvetica, we can update the `body` font family in the theme

@chrisunderdown thoughts/comments?

Reply:
Okay, I'll then update theme files to use body font